### PR TITLE
Fix #1546 Add local host name

### LINKED ISSUE #1546: Darwin naming: as per scutil terminology, LocalHostName and ComputerName
The `system_info`'s hostname entry is accurate in the traditional sense, but admins may want to know the other two pieces of naming criteria services may use (LocalHostName for bonjour/zeroIPconf discovery, and ComputerName for Apple Remote Desktop-style 'friendly' naming). This is usually retrieved with `scutil --get` and the other two name types, e.g.:  

```
odmit62:~ abanks$ scutil --get ComputerName
bar
odmit62:~ abanks$ scutil --get LocalHostName
foo
```

(And obviously I'm using the default prompt, which includes that distinct hostname)

### PR DESCRIPTION
**Pull request by jedi22**
Adding local host name which was referenced in an issue a while back.

### CODE REVIEW

**Comment by osqueryer** [comment]
:-1: The commit 7e535ac3d101c3583feecec0f36f9e7985dc6953 (Job results: [4962](https://jenkins.osquery.io/job/osqueryPullRequestLinux/4962/)) failed one or more tests (**Linux**).

**Comment by poppyseedplehzr** [comment]
This issue fixes #1546

**Comment by poppyseedplehzr** [comment]
This looks alright to me, but is there really no equivalent on linux/Windows for this behavior? Any chance you could drop some output of your query in the description, and I can see if there'd be any logical equivalent on windows? Adding the `hostname` to another column seems like extraneous output to me :\

**Comment by jedi22** [comment]
```SQL
osquery> select hostname, computer_name, local_host_name from system_info;
       hostname = obelisk-mbp5
  computer_name = a_computer_name
local_host_name = morenamesnounderscores
osquery>
```

**Comment by jedi22** [comment]
Using `sudo scutil --set LocalHostName morenamesnounderscores` to set the name manually.

File: specs/system_info.table [hunk 1]
```diff
@@ -14,5 +14,6 @@ schema([
     Column("hardware_version", TEXT, "Hardware or board version"),
     Column("hardware_serial", TEXT, "Device or board serial number"),
     Column("computer_name", TEXT, "Friendly computer name (optional)"),
+    Column("local_host_name", TEXT, "Local hostname (optional)"),
```
Comment (theopolis) [inline specs/system_info.table]: Maybe `local_hostname`?

**Pull request merged**